### ISSUE TITLE
Remove Symbol description

### ISSUE THREAD

**Issue by boopathi** [babel/babili#512]
```js
Symbol("description");

// out
Symbol();
```

Also, `Symbol()` can be treated as a Side-effect free fn call, and can be dead-code eliminated.

Ref: #510

**Comment by loganfsmyth** [comment]
Should this one be marked unsafe? Technically someone could be relying on the `.toString()` output.

**Comment by boopathi** [comment]
Yeah. When it is included, it should be included behind an option. 

Right now, the unsafe grouping is removed and options are flattened, and we need to re-think as to what options to gather under this one option - `unsafe`.
